tests/test_fetch.py

```diff
@@ -38,85 +39,60 @@
 )
 from conda.models.channel import Channel
 
+if TYPE_CHECKING:
+    from typing import Any, Callable
+
+    from pytest import MonkeyPatch
+
 
 @pytest.mark.integration
-def test_download_connectionerror():
-    with env_var(
-        "CONDA_REMOTE_CONNECT_TIMEOUT_SECS",
-        1,
-        stack_callback=conda_tests_ctxt_mgmt_def_pol,
-    ):
-        with env_var(
-            "CONDA_REMOTE_READ_TIMEOUT_SECS",
-            1,
-            stack_callback=conda_tests_ctxt_mgmt_def_pol,
-        ):
-            with env_var(
-                "CONDA_REMOTE_MAX_RETRIES",
-                1,
-                stack_callback=conda_tests_ctxt_mgmt_def_pol,
-            ):
-                with pytest.raises(CondaHTTPError) as execinfo:
-                    url = "http://240.0.0.0/"
-                    msg = "Connection error:"
-                    download(url, mktemp())
-                    assert msg in str(execinfo)
```

Comment:
Yikes! Thanks for getting rid of this! 🧹

Reply:
Strange that with env_vars() was not used